tools/cordova/package-id-version-parser.js

```diff
@@ -0,0 +1,27 @@
+// Accepts a combined Cordova package ID + version string, then parses out
+// and returns the ID and version in a package details object.
+//
+// Example `packageIdAndVersion` formats:
+// some-cordova-plugin@1.0.0
+// @somescope/some-cordova-plugin@1.0.0
+exports.parse = packageIdAndVersion => {
+  const package = {};
+  if (packageIdAndVersion) {
+    const parts = packageIdAndVersion.match(
+      /^(@[^\/]*)?(\/)?([^@]*)(@)?(.*)?/
```

Comment:
```js
/^(@[^\/]*)?(\/)?([^@]*)(@)?(.*)?/
         ^
```

When the scope is present in this manner, is it required that there be at least a character after the `@`?  If so,`+` ("1+ occurrence") would be more appropriate here than `*` ("0+ occurrences").  Similar question about the other `*` cases.*

<sub>
*Winner not guaranteed. ;-)
</sub>

Reply:
Yes - also a good call! Changes coming.